web_src/js/features/comp/ReactionSelector.js

```diff
@@ -1,38 +1,45 @@
 import $ from 'jquery';
 import {POST} from '../../modules/fetch.js';
 
-export function initCompReactionSelector($parent) {
-  $parent.find(`.select-reaction .item.reaction, .comment-reaction-button`).on('click', async function (e) {
-    e.preventDefault();
+export function initCompReactionSelector() {
+  const containers = document.querySelectorAll('.comment-list, .diff-file-body');
+  if (!containers.length) return;
```

Comment:
```suggestion
```

No need to check the length. `for-loop` on an empty list just does nothing.

Reply:
Done in https://github.com/go-gitea/gitea/pull/30453/commits/456ea0f7fdbc7f2c7f2bf111577cd79822bf12c6.